debian/control

```diff
@@ -1,4 +1,4 @@
-Source: libxfont1
+Source: libxfont-1.5.2
```

Comment:
I looked into Synaptic now and it seems to be no problem to use shorter package names.
Personally, I would call the source page `libxfont1`, since the major version 1 is inside.